packages/linter/src/rules/ModuleRuntimeUsed.ts

```diff
@@ -0,0 +1,27 @@
+import { Context } from "../index";
+import { Rule } from "../rule";
+
+export class ModuleRuntimeUsed extends Rule {
+  run({ $ }: Context) {
+    const isTransformed = $("html[transformed='self;v=1']").length > 0;
+    if (!isTransformed) {
+      return this.pass();
+    }
+    const isModuleVersion =
+      $("script[type='module'][src='https://cdn.ampproject.org/v0.mjs']")
```

Comment:
Just check for `/v0.mjs` to support runtime self-hosting in the future.

Reply:
done